ocs_ci/ocs/resources/cbt_metadata.py

```
@@ -0,0 +1,869 @@
+        self._created_ns_resources.append(sa)
+
+        cluster_res, ns_res = create_cbt_rbac(self.namespace, self.service_account)
+        self._created_cluster_resources.extend(cluster_res)
+        self._created_ns_resources.extend(ns_res)
+
+        self._create_ca_cert_configmap()
+        self._deploy_lister_pod()
+
+    # ---- ConfigMap ------------------------------------------------
+
+    def _read_configmap(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_read_configmap`

Add return type annotation: `None`

(ANN202)